IBPSA/Fluid/HeatPumps/ModularReversible/BaseClasses/PartialReversibleRefrigerantMachine.mo

```diff
@@ -0,0 +1,669 @@
+            {-102,30}})));
+
+  Modelica.Blocks.Interfaces.RealInput TEvaAmb(final unit="K", displayUnit="degC")
+    if use_evaCap and not use_busConOnl
+    "Ambient temperature on the evaporator side" annotation (Placement(
+        transformation(
+        extent={{10,-10},{-10,10}},
+        rotation=180,
+        origin={-150,-130}),iconTransformation(extent={{10,-10},{-10,10}},
+          origin={-112,-90},
+        rotation=180)));
+  Modelica.Blocks.Interfaces.RealInput TConAmb(final unit="K", displayUnit="degC")
+    if use_conCap and not use_busConOnl
+    "Ambient temperature on the condenser side" annotation (Placement(
+        transformation(
+        extent={{-10,10},{10,-10}},
+        rotation=0,
+        origin={-152,130}),iconTransformation(
+        extent={{-10,10},{10,-10}},
+        rotation=0,
+        origin={-112,90})));
+
+  IBPSA.Fluid.Sensors.MassFlowRate mEva_flow(redeclare final package Medium =
+        MediumEva, final allowFlowReversal=allowFlowReversalEva)
+    "Mass flow sensor at the evaporator" annotation (Placement(transformation(
+        origin={70,-60},
+        extent={{10,-10},{-10,10}},
+        rotation=0)));
+  IBPSA.Fluid.Sensors.MassFlowRate mCon_flow(final allowFlowReversal=
+        allowFlowReversalEva, redeclare final package Medium = MediumCon)
+    "Mass flow sensor at the evaporator" annotation (Placement(transformation(
+        origin={-50,100},
+        extent={{-10,10},{10,-10}},
+        rotation=0)));
+
+  Modelica.Blocks.Logical.Hysteresis hys(
+    final uLow=Modelica.Constants.eps,
+    final uHigh=ySet_small,
+    final pre_y_start=false) "Use default ySet value" annotation (Placement(
+        transformation(extent={{10,10},{-10,-10}}, rotation=180,
+        origin={-110,-90})));
+
+  RefrigerantCycleInertia refCycIneCon "Inertia model for condenser side"
+      annotation(Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=90,
+        origin={0,50})));
+  RefrigerantCycleInertia refCycIneEva "Inertia model for evaporator side"
+      annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=270,
+        origin={0,-50})));
+
+  IBPSA.Fluid.HeatPumps.ModularReversible.BaseClasses.InflowTemperatureSensor senTConIn(
+    final y=MediumCon.temperature(MediumCon.setState_phX(
+        port_a1.p,
+        inStream(port_a1.h_outflow),
+        inStream(port_a1.Xi_outflow))))
+    "Real expression for condenser inlet temperature" annotation (Placement(
+        transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=0,
+        origin={-98,32})));
+  IBPSA.Fluid.HeatPumps.ModularReversible.BaseClasses.InflowTemperatureSensor senTEvaIn(
+    final y=MediumEva.temperature(MediumEva.setState_phX(
+        port_a2.p,
+        inStream(port_a2.h_outflow),
+        inStream(port_a2.Xi_outflow))))
+    "Real expression for evaporator inlet temperature" annotation (Placement(
```

Comment:
This was a request by Michael, as we don't want to add dynamic equations but instead just get the fluid temperature. Maybe the term `sensor` is misleading. This class is just a `realExpression` with a unit pre-defined, so that the expandable bus also gets the correct unit.